packages/base-driver/test/e2e/express/server.e2e.spec.js

```diff
@@ -3,8 +3,19 @@ import axios from 'axios';
 import {createSandbox} from 'sinon';
 import B from 'bluebird';
 import _ from 'lodash';
+import { exec } from 'teen_process';
+import https from 'node:https';
 import {TEST_HOST, getTestPort} from '@appium/driver-test-support';
 
+async function generateCertificate (certPath, keyPath) {
```

Comment:
this test will not run on windows i'm assuming?

Reply:
it will only if openssl is installed and present in PATH